MozziGuts.h

```diff
@@ -136,7 +136,15 @@ are called.
 */
 void audioHook();
 
+/** @ingroup analog
+
+See getAudioInput(). The template parameter specifies the desired value range in bits. */
+template<byte RES> int16_t getAudioInput();
```

Comment:
I might be missing something, but I would have expected everything to be `uint16_t` and not `int16_t`: analogReads are always positive.

Reply:
Good point. This used to be int, and I figured I'd define that more precisely, while at it. Noteably, standard analogRead() returns int. However using int16_t (or int on AVR) would get us into trouble when requesting 16 bits. I  changed this to uint16_t

Reply:
I think this one escaped :P